exploration/expression-attributes.md

```diff
@@ -27,17 +28,31 @@ Function options may influence the resolution, selection, and formatting of anno
 These provide a great solution for options like `minFractionDigits`, `dateStyle`,
 or other similar factors that influence the formatted result.
 
-However, this single bag of options is not appropriate in all cases,
-in particular for attributes that pertain to the expression as a selector or a placeholder.
-For example, many of the [XLIFF 2 inline element] attributes don't really make sense as function options.
+Such options naturally correspond to function arguments or builder-style function constructors.
+Each option is specific to the associated API.
+Users (message authors, translators, developers) should not expect
+that a given option name has the same meaning between functions
+or that its behavior stays the same.
```

Comment:
I think this is backwards: if the learning curve is to be manageable, we want functions to be consistent. Overriding the locale or suppressing localization (to use two examples) should work the same everywhere rather than being idiosyncratic for each MF2 implementation **_and_** each function withing those implementations.

```suggestion
Message authors such as translators and developers want consistent ways to do common tasks, 
such as providing hints to translation tools or overriding the locale,
but, unless MessageFormat provides otherwise, cannot rely on implementations
to consistently implement these.
```